### DISCUSSION TITLE
Checking number length in json response body

### DISCUSSION THREAD

**Discussion by vladparl** [pactumjs/pactum#114] [Q&A] [answered]
Hello guys, 

I am using cucumber boilerplate project with pactumjs. I have response body like
{
"page": {
        "page": 1,
        "pageSize": 10,
        "totalItems": 41739
    }
}
and I need to check totalItems > someNumber in Then step. I am wondering how to write step definition for that. 

Any help will be much appreciated.

Here are all steps and I cannot find something similar:
https://github.com/pactumjs/pactum-cucumber-boilerplate/blob/master/features/support/steps.js

**Comment by ASaiAnudeep** [comment, marked as answer]
To compare dynamic values in JSON response, we can use `expectJsonLike` method which is mapped to the following step definition
 
```js
Then(/^I expect response should have a json like$/, function (json) {
  spec.response().should.have.jsonLike(JSON.parse(json));
});
```

Use [Assert Expressions](https://pactumjs.github.io/#/response-validation?id=assert-expressions) to easily solve the problem.

```gherkin
Then I expect response should have a json like
"""
{
  "page": {
    "totalItems": "$V > 41739"
  }
}
"""
```

**Reply by vladparl** [reply]
thank you ASaiAnudeep, works like a charm :)
